x-pack/test/ingest_manager_api_integration/config.ts

```diff
@@ -10,6 +10,11 @@ import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
 import { defineDockerServersConfig } from '@kbn/test';
 import { services } from '../api_integration/services';
 
+// Docker image to use for Ingest Manager API integration tests.
+// This hash comes from the commit hash here: https://github.com/elastic/package-storage/commit/48f3935a72b0c5aacc6fec8ef36d559b089a238b
+export const dockerImage =
```

Comment:
exposing this so the endpoint tests can import it.

Reply:
👍

Reply:
I feel like we should note somewhere to bump this whenever there's a new release. Probably not here, but not sure where would be best

Reply:
Yeah good point. I can probably add something to the endpoint package repo. The reason we would bump this is if there was a new release of the endpoint package I would think.